sw/device/silicon_creator/lib/base/sec_mmio.h

```diff
@@ -109,6 +109,30 @@ OT_ASSERT_MEMBER_OFFSET(sec_mmio_ctx_t, expected_write_count, 1608);
 OT_ASSERT_MEMBER_OFFSET(sec_mmio_ctx_t, check_count, 1612);
 OT_ASSERT_SIZE(sec_mmio_ctx_t, 1616);  // Checked by linker script.
 
+// The sec_mmio_ctx is referenced here to be able to use it inside the
+// `SEC_MMIO_WRITE_INCREMENT()` macro.
+extern volatile sec_mmio_ctx_t sec_mmio_ctx;
+
+/**
+ * Increment the expected count of register writes by `value`.
+ *
+ * This macro must be used to increment the number of expected register writes
+ * before calling `sec_mmio_check_counters()`.
+ *
+ * @param value The expected write count increment.
+ */
+#define SEC_MMIO_WRITE_INCREMENT(value)           \
+  do {                                            \
+    sec_mmio_ctx.expected_write_count += (value); \
+  } while (0);
```

Comment:
Note for the future: don't include this trailing semi to force callers to include one.

Reply:
ack. Done.